tests/qchem/of_tests/test_active_space.py

```diff
@@ -1,10 +1,14 @@
 import os
+import sys
 
 import pytest
 
 from pennylane import qchem
 
-# TODO: Bring importorskip inside relevant tests.
+# TODO: Bring pytest skip to relevant tests.
+if not (3, 9) < sys.version_info < (3, 10):
+    pytest.skip(allow_module_level=True)
```

Comment:
I think you should be able to add a `of_tests/conftest.py` file and include this here. This will then apply to _all_ tests in the `of_tests` folder, without you having to duplicate this

Reply:
Some modules in the of_tests do not depend on openfermion and I do not want to exclude them. As we discussed previously, the whole of_tests need a reorganization and I have a TODO for it.  The OF-dependent tests will be concentrated in one module and can be easily skipped if needed (prefer to do it in a dedicated PR).

Reply:
:+1: